docs/index.md

```diff
@@ -2,6 +2,19 @@
 
 ![Architecture](images/oasis-core-high-level.svg)
 
+## Development Setup
+
+Here are some examples on how to set up the local build environment and prepare
+some test networks for local development of Oasis Core components.
```

Comment:
Just a small nitpick. Please, reflow for the 80 character line limit.

```suggestion
Here are instructions on how to set up the local build environment, run the tests and
some examples on how to prepare test networks for local development of Oasis
Core components.
```

Reply:
These should already be at the 80 line limit, otherwise linter would complain?

Reply:
Oh you mean your suggestion, nevermind :-)